src/providers/push/apns/apns.provider.ts

```diff
@@ -37,7 +36,8 @@ export class ApnsProvider implements Provider<any> {
       );
     }
 
-    if (message.receiver.to.length > 500) {
+    const messageReceiverLength = 500;
```

Comment:
It's not message receiver length, 500 is the max limit of it. message receiver length is the value we get in `message.receiver.to.length`.